src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,96 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
-        />
-      </span>
-    </p>
-  </form>
+import React from 'react';
+
+import { FilterBy } from '../../types/FilterBy';
+import { FilterByOptionsNames } from '../../types/FilterByOptionsNames';
+
+const filterByOptions = Object.values(FilterBy);
+
+const filterByOptionsNames: FilterByOptionsNames = filterByOptions.reduce(
+  (names, filterByOption) => {
+    const filterByOptionSplit = filterByOption.split('');
+    const nameFirstLetterUpperCased = filterByOptionSplit[0].toUpperCase();
+    const nameRemainingPart = filterByOption.slice(1);
+    const name = nameFirstLetterUpperCased + nameRemainingPart;
+
+    return {
+      ...names,
+      [filterByOption]: name,
+    };
+  },
+  {} as FilterByOptionsNames,
```

Comment:
Consider uppercase the first letter of each option instead of this logic, it may be simpler